Work around Linux 3.2 xsave/avx bug until Xapi is capable of doing per-vm cpuid feature masking

Signed-off-by: Andrew Cooper <andrew.cooper3@citrix.com>

diff --git a/xen/arch/x86/cpu/common.c b/xen/arch/x86/cpu/common.c
index 5c8d3c2..910f5d3 100644
--- a/xen/arch/x86/cpu/common.c
+++ b/xen/arch/x86/cpu/common.c
@@ -15,7 +15,7 @@
 
 #include "cpu.h"
 
-static bool_t __cpuinitdata use_xsave = 1;
+static bool_t __cpuinitdata use_xsave = 0;
 boolean_param("xsave", use_xsave);
 
 bool_t __devinitdata opt_arat = 1;
